specs/phase1/beacon-chain.md

```diff
@@ -1085,8 +1093,7 @@ def process_epoch(state: BeaconState) -> None:
     process_reveal_deadlines(state)
     process_challenge_deadlines(state)
     process_slashings(state)
-    process_final_updates(state)  # phase 0 final updates
-    process_phase_1_final_updates(state)
+    process_final_updates(state)
```

Comment:
Separating into phases made order dependency issues. modularized phase 0 final updates into sub-functions and put into one function here for ordering control